### DISCUSSION TITLE
Patch for MC-93826

### DISCUSSION THREAD

**Discussion by Mickey42302** [PurpurMC/Purpur#1592] [Feature Requests]
I have a question/suggestion. One of the issues in Java Edition that remain unresolved to this day is MC-93826 (https://bugs.mojang.com/browse/MC-93826). After doing some research, I found that DivineMC and other forks have a patch for this bug (https://github.com/DivineMC/DivineMC/blob/ver/1.21.1/patches/server/0006-Fix-MC-93826.patch).

Would it be possible for Purpur to add this patch to the Purpur repository?

**Comment by Krakenied** [comment]
The patch from DivineMC doesn't work (they add a field that they do not even write to) so they probably do not know how it works or they have another patch modifying this field
